app/src/main/java/com/kickstarter/ui/activities/DeepLinkActivity.kt

```diff
@@ -109,6 +112,24 @@ class DeepLinkActivity : AppCompatActivity() {
             .subscribe {
                 startPreLaunchProjectActivity(it.first, it.second, "DEEPLINK")
             }.addToDisposable(disposables)
+
+        viewModel.outputs.startProjectSurvey()
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe {
+                val uri = it.first
+                val isLoggedIn = it.second
+                val surveyResponse =
+                    SurveyResponse.builder().urls(
```

Comment:
We could move this logic to the VM, and emit as outputs `startSurveyResponseActivity` and `startLoginForSurveys`, and add test for both use cases for maintainance sake